sdk/keyvault/tests.yml

```diff
@@ -5,17 +5,11 @@ stages:
     parameters:
       ServiceDirectory: keyvault
       TimeoutInMinutes: 240
-      SupportedClouds: 'Public,UsGov,China,Canary'
+      SupportedClouds: 'Public,UsGov,China'
+      TestMode: 'LIVE'
```

Comment:
This is unnecessary as we already default to LIVE in the live tests (https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/stages/archetype-sdk-tests.yml#L59-L61).

Reply:
Fixed in the new version.